src/main/java/monitoringsystemturbo/history/TimelineJsonDeserializer.java

```diff
@@ -42,7 +42,7 @@ private void deserializePeriods() throws ClassNotFoundException {
                 default:
                     throw new ClassNotFoundException("Period subclass not found (" + periodClassName + ")");
             }
-            periods.add(period);
+            observableList.add(period);
```

Comment:
change name to be more meaningful, we know that it is a observableList, but nothing about why it exists and what it stores